docs/application/dotnet/guides/multimedia/media-recording.md

```diff
@@ -190,18 +190,25 @@ To record audio:
 
 To initialize the video recorder for use:
 
-1.  Create an instance of the [Tizen.Xamarin.Forms.Extension.MediaView](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Xamarin.Forms.Extension.MediaView.html) class, which provides a view of the camera on the screen.
+1.  Create an instance of the [Tizen.Multimedia.Camera](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Multimedia.Camera.html) class.
+    ```
+    var camera = new Camera(CameraDevice.Rear);
+    ```
 
-    Create an instance of the [Tizen.Multimedia.Camera](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Multimedia.Camera.html) class and set the display of the camera to the `NativeView` property of the `Tizen.Xamarin.Forms.Extension.MediaView` instance:
+2. To set the display on which the video is recorded, use the `Display` property of the `Tizen.Multimedia.Camera` class.
+
+    For example, to set the display on a Xamarin-based application, first create an instance of the custom renderer(ex. VideoView) based on VisualElementRenderer class, cast it to an instance of the [Tizen.Multimedia.MediaView](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Multimedia.MediaView.html) class, and finally set that instance as the `Display` property:
```

Comment:
@sabeena21 Shouldn't we change "ex." to "For example"?

Reply:
I'll fix it.